libs/output-mapping/src/Writer/Helper/TableColumnsHelper.php

```diff
@@ -16,33 +17,42 @@ public static function addMissingColumns(
         array $newTableConfiguration,
         string $backendType,
     ): void {
-        $missingColumnsData = [];
         $missingColumns = self::getMissingColumnsFromColumnMetadata($currentTableInfo, $newTableConfiguration);
+        if (!$missingColumns) {
+            $missingColumns = self::getMissingColumnsFromColumns($currentTableInfo, $newTableConfiguration);
+        }
 
+        $missingColumnsData = [];
         if ($currentTableInfo['isTyped'] === true) {
-            foreach ($newTableConfiguration['column_metadata'] as $columnName => $columnMetadata) {
-                $columnName = ColumnNameSanitizer::sanitize($columnName);
-
-                if (!in_array($columnName, $missingColumns)) {
-                    continue;
+            if (!empty($newTableConfiguration['column_metadata'])) {
+                foreach ($newTableConfiguration['column_metadata'] as $columnName => $columnMetadata) {
```

Comment:
A nemůže teďka nastat situace, že sloupec bude v misssingColumns a nebude v $newTableConfiguration['column_metadata'] ?

Reply:
Tohle jsem měl původně ošetřený. Ale pak jsem si říkal že bysme měli být striktní a brát v potaz že buď máš zadaný všechny metadata, nebo máš smůlu.

Ale jak nadtím teď přemýšlím potřetí tak máš vlastně pravdu. Můžu mít zadaný metadata jen pro něco. Takže to upravím tak jak jsem to měl v pracovní verzi a pošlu znova na review.

Reply:
Fixnuto